### ISSUE TITLE
Improve Speed for fixed cameras

### ISSUE THREAD

**Issue by Wenuka** [davisking/dlib#888]
Hi, is there any way to speed up the dlib correlation tracker when the image is taken from a fixed CCTV camera which gives a still image.

**Comment by davisking** [comment]
Isn't that always how it works?  One frame at a time?  I'm not sure I
understand what you are asking.

**Comment by Wenuka** [comment]
No, I mean is there any way to increase the efficiency/ speed, given that the background is same for every frame (coz video is coming from a fixed CCTV camera)? i.e Didn't it take more computational power to track an object in a changing environment? coz there we cant apply simple methods like background subtraction and etc...

Anyway, thank you very much for your quick response.

**Comment by davisking** [comment]
Yes, there are many ways to detect and/or track things in video.  Some are
faster than others and more or less appropriate for any given situation.
